### DISCUSSION TITLE
채널톡 도입 가능한가요?

### DISCUSSION THREAD

**Discussion by Choi-JJunho** [woowacourse-teams/2023-pium#394] [질문있어요]
<img width="292" alt="image" src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/4b2db167-3ac0-4889-bc18-0de376844015">

현재 마이페이지에서 문의하기가 구글 폼으로 가고있습니다.

<img width="1438" alt="image" src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/a74a042f-fac9-4fb1-b8af-0d1deb1e86b7">

예상 소요가 어떻게 될 지는 모르겠지만... 채널톡같은거 도입해보는건 어떤가요?

**Comment by yeonkkk** [comment]
저는 채널톡 고려해볼만 한 것 같습니다.
구글폼은 단방향 소통 방식이라 사용자가 답답함을 느낄수도 있을 것 같아요.
채널톡을 이용하면 반복적인 문의에 대한 처리도 효율적으로 할 수 있고, 양방향으로 소통할수도 있어서 좋을 것 같아요.

**Comment by WaiNaat** [comment]
하루 이내로 세팅이 가능할 만큼 간편하다면 해보면 좋을 것 같아요
